container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+HOMEDIR=/var/lib/tempest
+TEMPEST_DIR=$HOMEDIR/openshift
+
+pushd $HOMEDIR
+
+export OS_CLOUD=default
+
+TEMPEST_LIST=$HOMEDIR/
+if [ ! -z ${USE_EXTERNAL_FILES} ]; then
+    TEMPEST_LIST=$HOMEDIR/external_files/
+    mkdir -p $HOME/.config/openstack
+    cp ${TEMPEST_LIST}clouds.yaml $HOME/.config/openstack/clouds.yaml
+fi
+
+tempest init openshift
+
+pushd $TEMPEST_DIR
+
+discover-tempest-config --os-cloud $OS_CLOUD --debug --create identity.v3_endpoint_type public
+
+if [ -f ${TEMPEST_LIST}include.txt ]; then
```

Comment:
Generally another question would be if there should be any 'default' limits to what to run or not - empty exclude sounds good (to not fail on it missing), though include should be empty or include-all i guess (i mean for now having it just scoped to identity seems ok, but later that should be dropped/expanded).

Reply:
My bad, fixing in the next path.
If you pass --include and --exclude to tempest, you must pass something inside, otherwise it will fail (file missing or file is empyt in case of the include) and yes, for now just identity because keystone is the minimum service that can run in an openstack deployment.